modules/ROOT/pages/configure-acm.adoc

```diff
@@ -311,14 +313,10 @@ In this section, you create the profile for members of your community. Users tha
 Next you need to give permissions to your members' profile so that they can access APIs, create client applications and manage their details. To do this:
 
 . Select *Edit* in the *ACM Member User* profile page.
-. In the *Standard Object Permissions* section:
-.. Enable *Read* permissions for:
-... *Documents*
+. In the *Standard Object Permissions* section, enable *Read* permissions for: *Documents*
 . In the *Custom Object Permissions* section:
-.. Enable *Read* permissions for:
-... *CommunityApi*
-.. Enable *Read*, *Edit*, *Create* and *Delete* permissions for:
-... *SelfRegisterUserRequests*
+.. Enable *Read* permissions for: *CommunityApi*
```

Comment:
ditto colons

There must be some formatting thing that I am not understanding. Please fill me in, if so!

Reply:
This file was a long developer written Google doc, which Paige adapted into a docs file this summer under tight deadline pressure, and it has mostly been left alone since then. Originally each of the sentences of this form was a list, and some lists had only one item. I changed those to sentences in the format "Make a change to: Item". The incorrect use of bold is also from the dev doc. So it is a formatting thing that's difficult to understand, but there's no reason to keep it, and I will change those to the format "Make a change to Item."